sympy/vector/tests/test_parametricregion.py

```diff
@@ -0,0 +1,50 @@
+from sympy import sin, cos, pi, symbols
+from sympy.vector.coordsysrect import CoordSys3D
+from sympy.vector.parametricregion import ParametricRegion
+
+C = CoordSys3D('C')
+r, theta, phi, a, b, t = symbols('r theta phi a b t')
+
+def test_parametricregion():
+
+    point = ParametricRegion(3, 4)
```

Comment:
What about `ParametricRegion((), (3, 4))` to define a point? That is... first argument is the tuple of parameters (in this case an empty tuple), second one the mapping.

I think this would be more intuitive. What do you think?

Reply:
@Upabjojr How will we get the definition of base scalars in terms of parameters? Also,  if we fix the number of arguments to 2, we have to use nested tuples to define the bounds for more than 2 parameters
```python
ParametricRegion((r, theta), ((r, 0, 2), (theta, 0, pi))
```

Reply:
This one looks good to me. It is still not very simple but we have to keep some minimum parameters. They are essential for defining a parametric region.

Reply:
It's the same concept for integrals:
```python
Integral(x, x)  # ==> indefinite
Integral(x, (x, 0, 10))  # ==> definite
```

If you have a better API idea, you're welcome to suggest.